### DISCUSSION TITLE
2020/mass-remove-live-photo-videos/

### DISCUSSION THREAD

**Discussion by giscus** [mirontoli/chuvash-eu-comments#4] [General]
# 2020/mass-remove-live-photo-videos/

While importing images from my iPhone using Image Capture on my mac, I discovered that almost all pictures had corresponding videos. They had the same name, only the file extension was different:
IMG_2829.JPG IMG_2829.MOV For archiving I don’t to have live photo videos, that’s why I needed a script for that. I found a good start in an answer on SuperUser.stackexchange.com:
OSX Command line Find Duplicate Filenames with different extensions I altered it a bit to remove the corresponding .

https://chuvash.eu/2020/mass-remove-live-photo-videos/

**Comment by pavelrazuvalau** [comment]
Huge kudos to you! You just saved tons of time for me!
